test/integration/findings-api.v1.test.js

```diff
@@ -0,0 +1,945 @@
+const FindingsApiV1 = require('../../dist/findings-api/v1');
+const { IamAuthenticator } = require('ibm-security-advisor/auth');
```

Comment:
We should require `'ibm-cloud-sdk-core'` rather than SDK itself.

Perhaps `const { IamAuthenticator } = require('../../dist/auth');`